iphone/Classes/TiUIListView.m

```diff
@@ -2158,30 +2204,28 @@ - (void)presentSearchController:(UISearchController *)controller
   if (resultsSeparatorStyle) {
     [resultViewController.tableView setSeparatorStyle:[TiUtils intValue:resultsSeparatorStyle def:UITableViewCellSeparatorStyleSingleLine]];
   }
-
   // Presenting search controller on window holding controller
-  id proxy = [(TiViewProxy *)self.proxy parent];
-  while ([proxy isKindOfClass:[TiViewProxy class]] && ![proxy isKindOfClass:[TiWindowProxy class]]) {
-    proxy = [proxy parent];
+  if (!searchControllerPresenter) {
+    id proxy = [(TiViewProxy *)self.proxy parent];
+    while ([proxy isKindOfClass:[TiViewProxy class]] && ![proxy isKindOfClass:[TiWindowProxy class]]) {
+      proxy = [proxy parent];
+    }
+    if ([proxy isKindOfClass:[TiWindowProxy class]]) {
+      searchControllerPresenter = [proxy windowHoldingController];
+    } else {
+      searchControllerPresenter = [[TiApp app] controller];
+    }
   }
-  UIViewController *viewController = nil;
-  if ([proxy isKindOfClass:[TiWindowProxy class]]) {
-    viewController = [proxy windowHoldingController];
-  } else {
-    viewController = [[TiApp app] controller];
+  BOOL shouldAnimate = YES;
+  if ([TiUtils isIOS9OrGreater]) {
+    shouldAnimate = NO;
   }
```

Comment:
Why not `BOOL shouldAnimate = ![TiUtils isIOS9OrGreater];` ?

Reply:
Good idea! Done.